### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ptitzler**
Fixed typos, terminology and added clarifications. Delivers #5

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -166,10 +179,11 @@ $ aws s3 cp s3://results-audioset-classify/<your_training_id>/models/main/balanc
 
 ## 4. Upload evaluation notebook on Watson Studio
 
-1. Create a new project `Audioset Classification` on Watson Studio.
-2. Navigate to `Assets -> Notebooks` and click on `New notebook`.
-3. On the next screen click on `From file` and upload the [audioclassify_inference.ipynb](audioclassify_inference.ipynb) file. 
-4. Upload `final_weights.h5` (file which we downloaded in the previous step) and `eval.h5` to the object storage linked to Watson Studio. This can be done by navigating to to `assets->New data asset` or clicking on the icon on the right to popup the data upload GUI as shown in the screenshot below. 
+1. [Log in to Watson Studio](https://www.ibm.com/cloud/watson-studio). Create a free account if you don't have one yet.
+2. Create a new project `Audioset Classification` on Watson Studio.
```
Comment (bdwyer2) [inline README.md]:
```suggestion
2. Create a new project `Audioset Classification` in Watson Studio.
```

File: README.md [hunk 2]
```diff
@@ -57,22 +59,33 @@ When the reader has completed this Code Pattern, they will understand how to:
 * In addition setup your [AWS S3 command line](https://aws.amazon.com/cli/) which can be used to create buckets and/or add files to COS.
    * Export `AWS_ACCESS_KEY_ID` with your COS `access_key_id` and `AWS_SECRET_ACCESS_KEY` with your COS `secret_access_key`
 
-### Setup IBM CLI & ML CLI
+### Provision a Watson Machine Learning service instance
+
+* Log in to the [IBM Cloud](https://console.bluemix.net/). Sign up for a free account if you don't have one yet.
```
Comment (bdwyer2) [inline README.md:64]:
```suggestion
* Login to [IBM Cloud](https://console.bluemix.net/). Sign up for a free account if you don't have one yet.
```

File: README.md [hunk 3]
```diff
@@ -57,22 +59,33 @@ When the reader has completed this Code Pattern, they will understand how to:
...
+	* View the generated credentials and take note of the values for `instance`, `username`, `password`, and `url`. You will need this information when you configure your IBM Cloud CLI.	
+
+
+### Set up IBM CLI & ML CLI
 
 * Install [IBM Cloud CLI](https://console.bluemix.net/docs/cli/reference/bluemix_cli/get_started.html#getting-started)
-  * Login using `bx login` or `bx login --sso` if within IBM
+  * Login using `ibmcloud login` or `ibmcloud login --sso` if within IBM
 * Install [ML CLI Plugin](https://dataplatform.ibm.com/docs/content/analyze-data/ml_dlaas_environment.html)
   * After install, check if there is any plugins that need update
```
Comment (bdwyer2) [inline README.md]: I think this sentence needs to be re-written

File: README.md (hunk 2 above)
Comment (ptitzler) [reply README.md:64]: I followed IBM writing guidelines: http://w3.ibm.com/standards/terminology/cgi-bin/lookup?11=main+form&11=acronym%7Eabbreviation&11=prohibited&term=log+in&submit=Search&source=en&db_name=LOGOS&pickDB=LOGOS&template=details&ug=corporate&target_switch=none&exact=on

Relevant section:
```
log in

verb

    To connect to a computer system or network by entering identification and authentication information at the workstation.

    Usage note: Write as shown when used as a verb--two words.
```

Comment (bdwyer2) [reply README.md:64]: Okay, there is a `login` on line 76

File: README.md (hunk 3 above)
Comment (bdwyer2) [reply README.md]: Maybe
```suggestion
   * After installation check for plugin updates
```

Comment (ptitzler) [reply README.md]: how about this:

Verify that the latest plug-in version is installed: `ibmcloud plugin update machine-learning`

File: README.md (hunk 2 above)
Comment (ptitzler) [reply README.md:64]: Updated

File: README.md (hunk 3 above)
Comment (ptitzler) [reply README.md]: Done

**Review by bdwyer2 — APPROVED** [review]

**Pull request merged**